.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
```

Comment:
I upgraded Android Studio, this likely was automatically introduced for compatibility